spec/features/requests/request_spec.rb

```diff
@@ -238,19 +237,18 @@
           expect(confirm_email.html_part.body.to_s).to have_content("Please do not use disinfectant or cleaning product on books")
         end
 
-        it 'allow CAS patrons to request an available ReCAP item with Help Me' do
-          visit "/requests/#{voyager_id}"
-          expect(page).to have_content "Requests for ReCAP materials will be unavailable during a planned system update"
-          expect(page).to have_content 'Help Me Get It'
-        end
+        # it 'allow CAS patrons to request an available ReCAP item with Help Me' do
```

Comment:
@kevinreiss Are we planning to do this again soon?  If not we should just delete the test.